service/routes.py

```diff
@@ -34,7 +34,11 @@
 def index():
     """Root URL response"""
     return (
-        "Reminder: return some useful information in json format about the service here",
+        jsonify(
+            name="Order REST API Service",
+            version="1.0",
+            paths=url_for("list_orders", _external=True),
```

Comment:
Just thinking.. could we add the endpoints for more calls or just this would be good enough?

Reply:
It's for the current sprint, there will be better documentation with all calls later on.

Reply:
LGTM.